business_calendar.gemspec

```diff
@@ -22,6 +22,8 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.3"
   spec.add_development_dependency "rake"
+  spec.add_development_dependency "pry"
+  spec.add_development_dependency "pry-debugger"
   spec.add_development_dependency "rspec"
```

Comment:
Can you specify this to be rspec 3.0+? The new expect syntax is, IIRC, new to RSpec 3.